src/components/AccountDropdown/AccountDropdown.tsx

```diff
@@ -72,12 +77,13 @@ export const AccountDropdown: React.FC<AccountDropdownProps> = props => {
    * react on accountIds
    */
   useEffect(() => {
+    if (accountId) return setSelectedAccountId(accountId) // set if we are fed a prop of accountId
     if (!accountIds.length) return
-    const accountId = accountIds[0] // default to the first when we receive them
-    accountId !== selectedAccountId && setSelectedAccountId(accountId) // don't set to same thing again
+    const firstAccountId = accountIds[0] // default to the first when we receive them
+    setSelectedAccountId(firstAccountId) // don't set to same thing again
```

Comment:
This comment can probably go now that we've removed the `accountId !== selectedAccountId` check.